### ISSUE TITLE
log level 8 - breaks cluster creation

### ISSUE THREAD

**Issue by mjudeikis** [openshift/openshift-azure#1293]
if log level set to 8 service catalog cant start:
```
controller-manager-j8sfd   0/1       CrashLoopBackOff   5          7m
controller-manager-r48j5   0/1       CrashLoopBackOff   5          7m
controller-manager-s2cht   0/1       CrashLoopBackOff   5          7m
```
with error
```
I0315 17:25:31.960601       1 leaderelection.go:194] successfully acquired lease kube-service-catalog/service-catalog-controller-manager
I0315 17:25:31.961329       1 event.go:221] Event(v1.ObjectReference{Kind:"ConfigMap", Namespace:"kube-service-catalog", Name:"service-catalog-controller-manager", UID:"b365b219-4746-11e9-a47f-000d3a2aa4fb", APIVersion:"v1", ResourceVersion:"5327", FieldPath:""}): type: 'Normal' reason: 'LeaderElection' controller-manager-j8sfd-external-service-catalog-controller became leader
F0315 17:25:32.033332       1 controller_manager.go:237] error running controllers: failed to get api versions from server: failed to get supported resources from server: unable to retrieve the complete list of server APIs: servicecatalog.k8s.io/v1beta1: an error on the server ("unable to set dialer for kube-service-catalog/apiserver as rest transport is of type *transport.debuggingRoundTripper") has prevented the request from succeeding
```

/kind bug

**Comment by mjudeikis** [comment]
```
client).WaitForInfraServices() checking daemonset kube-service-catalog/apiserver 
INFO[2019-03-15T17:23:14Z]/home/mjudeiki/.gvm/pkgsets/go1.11/global/src/github.com/openshift/openshift-azure/pkg/cluster/kubeclient/ready.go:104 github.com/openshift/openshift-azure/pkg/cluster/kubeclient.(*kubeclient).WaitForInfraServices() checking daemonset kube-service-catalog/controller-manager 
```

**Comment by mjudeikis** [comment]
``` componentLogLevel:
  apiServer: 8
  controllerManager: 8
  node: 12 ```

**Comment by jim-minter** [comment]
@mjudeikis can you dig out the upstream bz for this - I am wondering if it is fixed in 3.11.98 and we can revert the validation

**Comment by mjudeikis** [comment]
will revert
